test/models/CommonModel.spec.ts

```diff
@@ -147,6 +147,18 @@ describe('CommonModel', function() {
     });
   });
   describe('mergeCommonModels', function() {
+    test('should handle recursive models', function() {
```

Comment:
I do not know what are you testing here. I expected assertion.

Reply:
Assertion added.